src/cmd/lint.js

```diff
@@ -5,11 +5,59 @@ import {FileFilter} from './build';
 
 const log = createLogger(__filename);
 
+// Flow types
+
+export type LinterLogLevel = 'debug' | 'fatal';
+export type LinterOutputType = 'text' | 'json';
+
+export type LinterCreatorParams = {
+  config: {
+    logLevel: LinterLogLevel,
```

Comment:
yeah, we can (I was a bit unsure if I prefer the explicit type name or just add a union type in the logLevel type definition).